tests/kernel/adamw_step.cc

```diff
@@ -285,31 +285,40 @@ void run_cpu_test(TestData<T>& data)
 }
 
 #ifdef NNTILE_USE_CUDA
+// Helper function to check CUDA errors
+void check_cuda_error(cudaError_t error, const char* message)
+{
+    if (error != cudaSuccess)
+    {
+        throw std::runtime_error(std::string(message) + ": " + cudaGetErrorString(error));
+    }
+}
+
 // Helper function to run CUDA test and verify results
 template<typename T, bool run_bench>
 void run_cuda_test(TestData<T>& data)
 {
     T *dev_grad, *dev_first_moment, *dev_second_moment, *dev_p;
-    cudaMalloc(&dev_grad, sizeof(T) * data.num_elems);
-    cudaMalloc(&dev_first_moment, sizeof(T) * data.num_elems);
-    cudaMalloc(&dev_second_moment, sizeof(T) * data.num_elems);
-    cudaMalloc(&dev_p, sizeof(T) * data.num_elems);
+    check_cuda_error(cudaMalloc(&dev_grad, sizeof(T) * data.num_elems), "cudaMalloc dev_grad");
+    check_cuda_error(cudaMalloc(&dev_first_moment, sizeof(T) * data.num_elems), "cudaMalloc dev_first_moment");
```

Comment:
The line is too long, need to make lines 79 characters or shorter in the entire file.